src/Server/Server.php

```diff
@@ -209,8 +217,10 @@ public function getUserEmail(TokenCredentials $tokenCredentials, bool $force = f
     /**
      * Get the user's screen name (username), if available.
      *
-     * @throws CredentialsException If a "bad response" is received by the server
```

Comment:
Again, this is an insight into the chain of methods that this calls. They throw an exception that this doesn’t catch. That is why this docblock was here. 

I’d suggest pulling up the project in php storm and it will provide insights into the decisions behind these.